### PR TITLE
Update client

### PR DESCRIPTION
**Pull request by ingdjason**
refactor client side and API to work with the new API (POSTGRES)
     - POSTGRE CRUD API
     - update getall in client side
    - Refactor 

Important file : 
App.jsx
postgresdb.js
app.js

### CODE REVIEW

File: db/postgresdb.js [hunk 1]
```diff
@@ -84,22 +94,52 @@ function savePostgreData(){
...
+  client.query(queryStr, [sellerID], (err, result) => {
+    if (err) {
+      callback(`Error  executing query, ${err.stack}`, null);
+    }
+    callback(null, result.rows);
+  });
+}
+
+let insertCommentReviewPostgres = (sellerID, name, avatar, comment, photocomment, item, itemphoto, rating, callback) => {
+  const queryStr = 'SELECT * FROM public.comment_review WHERE sellerid = $1 LIMIT 100';
```
Comment (j-gens) [inline db/postgresdb.js]: should this be a 'INSERT INTO ...' query?

File: db/postgresdb.js [hunk 2]
```diff
@@ -84,22 +94,52 @@ function savePostgreData(){
...
+    }
+    callback(null, result.rows);
+  });
+}
 
-let getAllPostgres = () => {
-  client.query('SELECT * FROM seller_info',  (err, result) => {
-    release();
+let updateCommentReviewPostgres = (sellerID, comment, ratings, callback) => {
+  const queryStr = 'SELECT * FROM public.comment_review WHERE sellerid = $1 LIMIT 100';
```
Comment (j-gens) [inline db/postgresdb.js]: should this be an 'UPDATE ...' query?

File: db/postgresdb.js (hunk 1 above)
Comment (ingdjason) [reply db/postgresdb.js]: yeah, just added the function but it's not complete yet

File: db/postgresdb.js (hunk 2 above)
Comment (ingdjason) [reply db/postgresdb.js]: Same thing ?

**Pull request merged**